brackend/tasks/tasks.py

```diff
@@ -35,7 +35,7 @@ def save_new_tournament(name, firebase_id):
         session.commit()
         current_app.logger.info("Users: %s", new_tourney.users)
         current_app.logger.info("Tournaments: %s", user.tournaments)
-        return new_tourney
+        return new_tourney.to_json()
```

Comment:
I guess doing this does clean up the api-layer above. I do worry that as the application gets more complex, and maybe more code exists that isn't just CRUD type stuff on the api layer, returning and passing around json instead of the `Tournament` object could get a little annoying, but that might be a far off concern, and might never really come to anything

Reply:
well, the real thing is that if we call to_json on a tournament object without a session we get some weird session not attached error  when trying to get all the brackets

maybe this is fixed by adding that subquery loading stuff everywhere?

Reply:
Ah I see, yeah I think that's something we should fix by doing the subquery load. There may be a better way than that, but would need to look into it more